lib/pages/main/tab_settings/tab_settings.dart

```diff
@@ -0,0 +1,142 @@
+      ),
+      body: Padding(
+        padding: ThemeEdgeInsets.pageInsets,
+        child: Column(
+          children: [
+            Expanded(
+              child: SingleChildScrollView(
+                child: Column(
+                  children: [
+                    SettingsListTile(
+                      prefix: const Icon(
+                        Icons.lock,
+                        color: LightThemeColors.textColorSecondary,
+                      ),
+                      title: l10n.settingsLockWallet,
+                      suffix: const SizedBox.shrink(),
+                      onPressed: () {
+                        appStore.reportGlobalError("");
+                        appStore.reportGlobalNotification("");
+                        appStore.setCurrentTabIndex(
+                            0); // so after unlock, it goes to Home
+                        appStore.signOut();
+                        appStore.checkWalletIsInitialized();
+                        timedController.stopFetchTimers();
+                        context.go('/signInNoAuth');
+                      },
+                    ),
+                    SettingsListTile(
+                      prefix: SvgPicture.asset(AppIcons.autoLock, height: 20),
+                      title: l10n.settingsLabelAutlock,
+                      path: AutoLockSettings(),
+                    ),
+                    SettingsListTile(
+                      prefix: SvgPicture.asset(AppIcons.export, height: 20),
+                      title: l10n.settingsLabelExportWalletVaultFile,
+                      path: const ExportWalletVault(),
+                    ),
+                    SettingsListTile(
+                      prefix:
+                          SvgPicture.asset(AppIcons.changePassword, height: 20),
+                      title: l10n.settingsLabelChangePassword,
+                      path: const ChangePassword(),
+                    ),
+                    SettingsListTile(
+                      prefix: settingsUnlockIcon,
+                      title: settingsUnlockLabel,
+                      path: const ManageBiometrics(),
+                    ),
+                    SettingsListTile(
+                      prefix:
+                          SvgPicture.asset(AppIcons.walletConnect, height: 20),
+                      title: l10n.settingsLabelWalletConnect,
+                      onPressed: () {},
+                    ),
+                    SettingsListTile(
+                      prefix: SvgPicture.asset(AppIcons.community, height: 20),
+                      title: l10n.settingsLabelJoinCommunity,
+                      path: const JoinCommunity(),
+                    ),
+                    SettingsListTile(
+                      prefix:
+                          SvgPicture.asset(AppIcons.privacyPolicy, height: 20),
+                      title: l10n.settingsLabelPrivacyPolicy,
+                      onPressed: () {},
+                      suffix:
```

Comment:
It seems to me that the action has not be implemented:
"Would link to https://qubic.org/Privacy-policy" as answered by Oreo in the Figma file.